### ISSUE TITLE
Both the VGA fonts for terminal and Helvetica fonts don't support slavic characters

### ISSUE THREAD

**Issue by SolsticeSpectrum** [grassmunk/Chicago95#314]
I am on latest Xubuntu which I unfortunately installed in Czech language because I didn't know this would be an issue later, problem with Xubuntu is that once you install it in specific language, you can't just switch back to English as all the folders and stuff already use czech translations. So maybe it would be nice if those fonts did support slavic characters. 

I am talking about: "ěščřžýáíé"

![image](https://github.com/grassmunk/Chicago95/assets/38611351/8413c8a5-9ef5-432a-a5d6-603085c20435)
![image](https://github.com/grassmunk/Chicago95/assets/38611351/246a7e15-accb-4eb4-856f-739bde8f89f9)
![image](https://github.com/grassmunk/Chicago95/assets/38611351/43714b92-e2f0-4aa3-a127-7295a45d1241)
As you can see, in both titles and terminal, these characters look weird

**Comment by 1chan-level-journalist** [comment]
On topic of Helvetica font not including anything other than Latin and Cyrillic characters - it's been bothering me for awhile, too. 
I've been working on creating a massive compilation of various versions of MS Sans Serif in a single .otb file. It is very close to completion now. It's not quite ready for release yet, as I'm yet to finish Arabic and Thai languages, but I believe it should satisfy your use case already.
I just did a VirtualBox test install, here's what it looks like with the font installed:
![screenshot](https://github.com/grassmunk/Chicago95/assets/16210758/61ec485e-0349-4939-8f81-f0bb70f92c89)
Looks good to me.
[MSSansSerifRaster.zip](https://github.com/grassmunk/Chicago95/files/13753489/MSSansSerifRaster.zip)
The name of the font is "MS Sans Serif Raster". Try it out!

As for the VGA terminal font - try this one out: [VGA.zip](https://github.com/grassmunk/Chicago95/files/13753421/VGA.zip)
The name is "VGA Medium".

**Comment by DarkReaper231** [comment]
@1chan-level-journalist You are EPIC! Yes both the Raster and VGA fonts work like a charm! I tried MSSansSerif myself by copying it from my Windows installation but it didn't fix the issue and there were other issues, your version however works flawlessly! The VGA font works like a charm as well!

![image](https://github.com/grassmunk/Chicago95/assets/38611351/f55a80d9-2693-4024-ad13-00a050870bc6)
![image](https://github.com/grassmunk/Chicago95/assets/38611351/8e7c5d4e-0554-4baf-b9e6-0a9d11db1825)
Some titles are no longer bold however but I assume that's how it's meant to be in the first place? 
![image](https://github.com/grassmunk/Chicago95/assets/38611351/b35b2731-7bdb-4d02-8ce4-8ca7c87702bf)

**Comment by DarkReaper231** [comment]
Also the `í` character in the bold variant has no space next to it
![image](https://github.com/grassmunk/Chicago95/assets/38611351/09f67524-554b-4858-840d-20c86769b7cc)

**Comment by DarkReaper231** [comment]
Okay the solution to fix the titles not being bold was just setting the title font to the bold one in window manager settings lol, my bad
![image](https://github.com/grassmunk/Chicago95/assets/38611351/ad5cb198-cebe-420f-a255-c241b450863f)

again `í` character has padding issues in the bold one
![image](https://github.com/grassmunk/Chicago95/assets/38611351/94a7c4cd-24a9-469e-a6b2-4c939134d2dc)

**Comment by DarkReaper231** [comment]
![image](https://github.com/grassmunk/Chicago95/assets/38611351/6e79b1da-c55e-4455-a076-a2d4d43d0b44)
the `/` character should also have some more space behind it

**Comment by 1chan-level-journalist** [comment]
On topic of some symbols not looking right - I've had an issue with FontForge, where the .sfd for regular variant got corrupted at some point, where some characters turned into duplicates of other characters. Seems like bold variant was unaffected however.

As for the character kerning issue - that's another issue that's been plaguing me for awhile, and I have no idea what is causing this. I was copying everything exactly how it was in the original .FON files, and yet some characters just don't kern the same way as they do in the original. Maybe the issue lies with the .otb format itself? Regardless, the only way to fix this is to manually alter the kerning so that it looks right... when it allows me to do that, that is! There are times when the character kerning doesn't change no matter what I do to it - bold slash is one of them.
[MSSansSerifRaster.zip](https://github.com/grassmunk/Chicago95/files/13762620/MSSansSerifRaster.zip)

By the way if you use .otb fonts with Qt-based programs, some windows would look super-stretched. Like this: 
![screen](https://github.com/grassmunk/Chicago95/assets/16210758/d3044a2d-ae1c-4601-a436-3ccea17400c9)
Apparently it has something to do with how Qt handles .otb fonts. One way to avoid this is to convert the .otb font into .bdf format, then convert that into a .pcf format, and use that font instead - unlike GTK-based apps, Qt-based apps still can use .pcf fonts, and there is no such problem.
[MSSansSerifRasterPCF.zip](https://github.com/grassmunk/Chicago95/files/13762651/MSSansSerifRasterPCF.zip)

**Comment by DarkReaper231** [comment]
@1chan-level-journalist thank you very much, the new version fixed all the issues expect `/` but I can live with that for now
![image](https://github.com/grassmunk/Chicago95/assets/38611351/7dc2dcd5-4fe2-4967-833b-7a6a16c26eb6)
the `í` works fine now but I think you forgot `ï` and i with two dots above it, it doesn't have that spacing applied like `í` does

**Comment by 1chan-level-journalist** [comment]
How about this one?
[MSSansSerifRaster.zip](https://github.com/grassmunk/Chicago95/files/13766738/MSSansSerifRaster.zip)
[MSSansSerifRasterPCF.zip](https://github.com/grassmunk/Chicago95/files/13766795/MSSansSerifRasterPCF.zip)

**Comment by Dejweed** [comment]
Awesome work! Could you make a git repository or something similar, so you don't have to share files every time?

**Comment by 1chan-level-journalist** [comment]
Yeah, I'll make one once I finish all the languages, which shouldn't take too long.

**Comment by 1chan-level-journalist** [comment]
I just discovered a workaround to "font kerns not applying to certain symbols" issue!
While FontForge still refuses to cooperate with me on this, I can just edit the .sfd in a text editor manually, and then generate font from that.
[MSSansSerifRaster.zip](https://github.com/grassmunk/Chicago95/files/13793537/MSSansSerifRaster.zip)
[MSSansSerifRasterPCF.zip](https://github.com/grassmunk/Chicago95/files/13793587/MSSansSerifRasterPCF.zip)
The slash should look fine now. Try it out!

**Comment by 1chan-level-journalist** [comment]
The font is ready.
https://github.com/1chan-level-journalist/MS-Sans-Serif-Raster

**Comment by DarkReaper231** [comment]
@1chan-level-journalist Sorry I got busy with life but I am back on Chicago and I downloaded the latest update of the font and everything seems to be fine, good job!

**Comment by Dejweed** [comment]
> The font is ready. https://github.com/1chan-level-journalist/MS-Sans-Serif-Raster

Nice! Could you also add the VGA font?
